pypesto/visualize/parameters.py

```diff
@@ -115,6 +115,68 @@ def parameters(
     return ax
 
 
+def parameter_hist(
+        result: Result,
+        parameter_name: str,
```

Comment:
Also allows for lists? :thinking: ?

Reply:
Actually, lists are now allowed at the moment